pretext/GeneralIntro/WPToUnderstandaProgramChangeIt.ptx

```diff
@@ -1,15 +1,26 @@
 <?xml version="1.0"?>
 <section xml:id="general-intro_to-understand-a-program-change-it">
   <title>To Understand a Program, Change It!</title>
-  <p>To check your understanding or your predictions, you will run a program.</p>
+  <p>To check your understanding or your predictions, <alert>always run the program.</alert> This seems like 
+  an obvious step a programmer would always make, but it is something that novice programmers too often overlook doing.</p>
   <p>To check your understanding about the state of variables before your code snippet runs, add
-    diagnostic print statements that print out the types and values of variables. Add these print
-    statements just <em>before</em> the code snippet you are trying to understand.</p>
+    diagnostic print statements that print out the types and values of variables. Sometimes referred to as 'echo prints', add these print
+    statements just <em>before</em> and <em>after</em> the code snippet you are trying to understand.</p>
+
+<program language="python">
+    <input>
+      x = 4
+      y = 7
+      print(x, type(x), y, type(y)) # echo print
+      y = x + 0.2 ;print(type(y), "y=",y) # diagnostic print on same line
```

Comment:
Do we want to show them semicolons? I think this is the only time this is used and it's not very common.

Reply:
Again, an approach I used to use to support novice students with their debugging. 

Not necessary what with VS Codes debugging features.